### ISSUE TITLE
Not working for GoPro 6

### ISSUE THREAD

**Issue by springerarthur** [KonradIT/GoProStream#25]
Your python script work very well with the gopro 4. But now I try it with a gopro 6 black with the latest firmware.

Your script write only some logs but did not start the stream:

```
branch HD4
HERO6 Black
HD6.01.02.01.00
UDP target IP: 10.5.5.9
UDP target port: 8554
message: _GPHD_:0:0:2:0.000000

Recording on camera: False
```

When I try to execute the ffplay command direct, than I only get a "**nan**".
`ffplay -fflags nobuffer -f:v mpegts -probesize 8192 udp://:8554`
`ffplay version N-90061-gfb580731c1 Copyright (c) 2003-2018 the FFmpeg developers
  built with gcc 7.2.0 (GCC)
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-bzlib --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libmfx --enable-amf --enable-cuda --enable-cuvid --enable-d3d11va --enable-nvenc --enable-dxva2 --enable-avisynth
  libavutil      56.  7.101 / 56.  7.101
  libavcodec     58. 11.101 / 58. 11.101
  libavformat    58.  9.100 / 58.  9.100
  libavdevice    58.  1.100 / 58.  1.100
  libavfilter     7. 12.100 /  7. 12.100
  libswscale      5.  0.101 /  5.  0.101
  libswresample   3.  0.101 /  3.  0.101
  libpostproc    55.  0.100 / 55.  0.100`
    
`nan    :  0.000 fd=   0 aq=    0KB vq=    0KB sq=    0B f=0/0`

All other commands like switching the mode or capture a photo work very well by send the appropriated HTTTP get command.
When I'm getting the [http://10.5.5.9/gp/gpControl/execute?p1=gpStream&a1=proto_v2&c1=restart](http://10.5.5.9/gp/gpControl/execute?p1=gpStream&a1=proto_v2&c1=restart) than I'm getting the correct response **{"status":"0" }**

Is there a way to get more informations from ffplay?

Do you know what I'm doing wrong? Or is there a problem with the latest gopro firmware.

**Comment by KonradIT** [comment]
That is weird. Try downloading the script again as-is and run it.

**Comment by springerarthur** [comment]
I tried it now on an other notebook with the latest version of ffmpeg and of your script. But i've got the same result. Your script did nothing, excepted the written logs.When I try to run the ffplay command , then I also get only a nan message.

Maybe there is a wrong GoPro setting? 

Do you have any idea how I can debug the problem?

**Comment by KonradIT** [comment]
are you using Windows?

**Comment by ciancu-gpsw** [comment]
Is there a problem with windows platform? I tried to get the stream from a hero 5 session, and failed.

**Comment by KonradIT** [comment]
yes there is @ciancu-gpsw Windows blocks UDP traffic so you need to fix the firewall.

**Comment by springerarthur** [comment]
hmm...
I tried it again on a windows machine and disabling the firewall -> Not working
I tried it now on a linux (debian) -> Still not working

It's realy weird. I think that the problem will be the gopro.

Do you have any other idea how I can fix the problem? Otherwise I had to buy a new gopro and had to test it with the new gopro.

**Comment by KonradIT** [comment]
Hmm, do you know how to sniff UDP data from a phone running the GoPro app? might buy a Hero6 myself.

**Comment by runings** [comment]
Had the same problem on Windows 10, seems to be a problem with ffplay!

Changed the the last part of the ffplay command in the python script to:

... udp://127.0.0.1:8554 
or
... udp://10.5.5.100:8554

and it works just fine :-)

**Comment by gka748** [comment]
can you specify which line exactly on the script. I am using windows 10 @runings

**Comment by runings** [comment]
Line 94: 
From -->
subprocess.Popen("ffplay " + loglevel_verbose + " -fflags nobuffer -f:v mpegts -probesize 8192 udp://:8554", shell=True)
To -->
subprocess.Popen("ffplay " + loglevel_verbose + " -fflags nobuffer -f:v mpegts -probesize 8192 udp://127.0.0.1:8554", shell=True)

**Comment by KonradIT** [comment]
@runings @gka748 change pushed to the repo :) thanks! Works on Linux as well.

**Comment by gka748** [comment]
thank you @runings  it's working

**Comment by yarodevuci** [comment]
I have black screen on Here 6 and 7
Using mac @KonradIT
